### PR TITLE
Fix null reviews on sorting by reviews

### PR DESCRIPTION
**Pull request by impmihai**
Added a filter in order to move null review count
to the bottom of the list. The filter can be found
here : http://stackoverflow.com/a/27963602

### CODE REVIEW

File: resources/js/app.js [hunk 1]
```diff
@@ -239,6 +268,7 @@ app.directive('getinvolved', ['$http', function ($http) {
     .then(function (data) {
      $scope.$get_involved_storage.contributors_data = data["data"]
      self.contributors = data["data"]
+     console.log(data["data"]);
```
Comment (Mariatta) [inline resources/js/app.js]: Remove the logging :)

**Comment by Mariatta** [comment]
Git commit message footer should reference the issue being fixed.

Something like:
`Fixes https://github.com/coala/landing-frontend/issues/127`

**Comment by Mariatta** [comment]
Commit shortlog should be something like:

getinvolved.html: Fix the ordering based on reviews

Contributors with null reviews were being displayed on top of the list.
They are supposed to be shown at the bottom of the list.

Fixes https://github.com/coala/landing-frontend/issues/127

Please see coals.io/commit for more info about commit message.

Thanks.

File: partials/tabs/getinvolved.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
     </div>
     <div class="container">
         <div class="row showcase-gi">
-            <div  ng-repeat="member in gic.contributors | toArray | orderBy : 'reviews' : true | limitTo:totalDisplayed" class="card card-main showcase-card-gi">
+            <div  ng-repeat="member in gic.contributors | toArray | orderBy : 'reviews' : true | orderEmpty:'reviews':'toBottom' | limitTo:totalDisplayed" class="card card-main showcase-card-gi">
```
Comment (Mariatta) [inline partials/tabs/getinvolved.html]: Would be nice to break this apart so the line isn't too long :)

**Comment by gkrishnan724** [comment]
You forgot a space between `displayed` and `on` in the commit message :stuck_out_tongue: 

Code looks good to me  :+1:

**Comment by hemangsk** [comment]
Works perfect, tested locally as well 👍

**Comment by Mariatta** [comment]
ack a99aac1

**Comment by Mariatta** [comment]
@rultor merge

**Comment by Mariatta** [comment]
Thank you, @impmihai !!! :tada:

**Pull request merged**